OlympGuide.Application/Features/User/UserService.cs

```diff
@@ -87,6 +87,27 @@ public async Task<UserProfile> GetUserProfile(string accessToken)
             }
         }
 
+        public async Task<UserProfile> UpdateUser()
+        {
+            var token = _userContext.GetTokenFromCurrentUser();
+            if (string.IsNullOrEmpty(token))
+                throw new InvalidOperationException("Bearer token is emtpy or null");
+
+            var userInformations = await _authenticationProvider.GetUserInformations(token);
+            var userIdentifier = await _authenticationProvider.GetUserIdentifierFromToken(token);
+            var user = await _repository.GetByIdentifier(userIdentifier);
+            
+            user.Roles = userInformations.Roles;
+            user.Name = userInformations.Name;
+            user.Email = userInformations.Email;
+            user.DisplayName = userInformations.DisplayName;
+
+            _logger.LogInformation("User is udpated.");
```

Comment:
"user is updated" vor updateUser() und nacher "user has been updated"? Einer der Logs lügt